src/System.Runtime.Extensions/tests/System/BitConverter.cs

```diff
@@ -208,6 +208,24 @@ public static void RoundtripString()
         Assert.Equal(String.Empty, BitConverter.ToString(new byte[0]));
     }
 
+    [Fact]
+    public static void ToString_ByteArrayToLong_Throws()
+    {
+        byte[] arr;
+        try
+        {
+            arr = new byte[int.MaxValue / 3 + 1];
+        }
+        catch (OutOfMemoryException)
+        {
+            // Exit out of the test if we don't have an enough contiguous memory
```

Comment:
Just throwing this out there, not sure if we want to do this, but Xunit lets you throw something like an "Xunit.SkipException" to indicate a test was skipped. It might be appropriate in places like this, but then again we might want to save "Skips" for tests that we never tried to run (wrong platform, etc).